internal/pkg/service/common/etcdop/op/join.go

```diff
@@ -1,79 +0,0 @@
-package op
-
-import (
-	"context"
-
-	etcd "go.etcd.io/etcd/client/v3"
-)
-
-type JoinTo[R any] struct {
-	result *R
-	txn    *TxnOpDef
-}
```

Comment:
Ulohou `Join` operácie malo byť to, ak si chcel namapovať výsledky z TXN na nejaký typ.
Napr. získal si 3 etcd kluce a z toho si poskladal nejaku strukturu.
Funguje to jednoducho, ale nepouziva sa to, takze to zatial mazem, kym sa neukaze, ze to potrebujeme.